linux/keyman-config/keyman_config/ibus_util.py

```diff
@@ -11,11 +12,22 @@
 
 gi.require_version('IBus', '1.0')
 from gi.repository import IBus
+from pkg_resources import parse_version
 
 from keyman_config.gnome_keyboards_util import is_gnome_shell
 from keyman_config.gsettings import GSettings
 
 
+class IbusDaemon(Enum):
+    """
+    States of the ibus-daemon
+    """
+    RUNNING = 0,        'ibus-daemon is running'
+    NOT_RUNNING = 1,    'ibus-daemon is not running'
+    MORE_THAN_ONE = 2,  'Error: more than one instance of ibus-daemon is running'
+    ERROR = 3,          'Error: invalid/unknown user or exception'
```

Comment:
Do you want a separate error for unknown user / fail to SUDO?

Reply:
Done.